samza-core/src/main/java/org/apache/samza/zk/ZkUtils.java

```diff
@@ -190,7 +188,7 @@ public void publishJobModel(String jobModelVersion, JobModel jobModel) {
    * @return job model for this version
    */
   public JobModel getJobModel(String jobModelVersion) {
-    LOG.info("pid=" + processorId + "read the model ver=" + jobModelVersion + " from " + keyBuilder.getJobModelPath(jobModelVersion));
+    LOG.info("read the model ver=" + jobModelVersion + " from " + keyBuilder.getJobModelPath(jobModelVersion));
```

Comment:
Is it usually helpful to log the `processorId` in state transitions / reads like these? Could potentially be useful if there are multiple `StreamProcessor`s running?